src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import {Ellipsis} from '@gravity-ui/icons';
 import _memoize from 'lodash/memoize';
 
+import type {PopperPlacement} from '../../../../hooks/private';
```

Comment:
We should consume this type from `Popup` index

Reply:
That's right, it takes from `../hooks/private/index/usePopper/index/usePopper`

https://github.com/gravity-ui/uikit/blob/59eae6957ab0033cf26fe3197bbe23bffb31ef2f/src/hooks/private/usePopper/usePopper.ts#L9

and `PopperPlacement` is used in `PopperProps` and `PopupProps` extends from `PopperProps`